codecov.yml

```diff
@@ -30,3 +30,8 @@ comment: false
 # https://docs.codecov.io/docs/flags
 # TODO: split out test coverage for API, SDK, UI, website?
 # flags:
+
+ignore:
+  - "agent/bindata_assetfs.go"
+  - "vendor/**/*"
```

Comment:
This is fine to have but isn't needed since we don't push coverage reports from this folder up so it wouldn't show up anyway.